helm/values.yaml

```diff
@@ -41,11 +41,6 @@ skipInstall: true
 customPostInitScripts:
   install-openg2p-modules.sh: |
     #!/bin/bash
-    rm -rf /tmp/openg2p-erp*
-    curl -L -o /tmp/openg2p-erp-community-addon.zip $OPENG2P_CA_REPO/archive/$OPENG2P_CA_BRANCH.zip &&
-    curl -L -o /tmp/openg2p-erp.zip $OPENG2P_ERP_REPO/archive/$OPENG2P_ERP_BRANCH.zip &&
-    unzip /tmp/openg2p-erp-community-addon.zip -d /tmp &&
-    unzip /tmp/openg2p-erp.zip -d /tmp &&
     cp -rf /tmp/openg2p-erp*/* /bitnami/odoo/addons
```

Comment:
These lines couldn't be moved?

Reply:
The below lines are for installing modules.